### PR TITLE
Update README.md for summer 2023

### PR DESCRIPTION
**Pull request by chendaniely**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,7 +162,7 @@ Students **must be present** at the invigilation venue (in class, examination ce
 
 Students who will miss a quiz **must provide a self-declaration of academic concession prior to the quiz** (see Canvas homepage for the academic concession form) and make arrangements with the Instructor. Failing to present a declaration within a reasonable timeframe before the quiz will result in a grade of zero.
 
-There will be **no extensions for the lecture and tutorial worksheets**; late assignments will receive a grade of zero. Instead, we will drop the lowest grade on tutorials and worksheets for the semester. However, if you have extenuating circumstances and require further accommodations, please contact the course coordinator. 
+There will be **no extensions for the lecture and tutorial worksheets**; late assignments will receive a grade of zero. Instead, we will drop the lowest grade on tutorials and worksheets for the semester. However, if you have extenuating circumstances and require further accommodations, please contact the course coordinator.
```
Comment (mgyliu) [inline README.md:165]: Should this say lowest 2 grades?

Comment (chendaniely) [reply README.md]: should be updated now

File: README.md [hunk 2]
```diff
@@ -65,25 +65,23 @@ By the end of the course, students will be able to:
...
-| All     |          TA                   |  Ding Ma         |             n/a              | Fridays 7 - 8 PM | Zoom |
-| All     |          TA                   |  Nour  Hanafi      |             n/a              | Wednesdays 12 - 1 PM | ESB 1045 |
-| All     |          TA                   |  Fares Burwag         |             n/a              | Fridays 6-7 PM | Zoom |
-| All     |          TA                   |  Edward Sobczak         |           n/a         | Wednesdays 6-7 PM  | Zoom |
-| All     |          TA                   |  Ali Mehrabian         |           n/a         | Thursdays 4-5 PM  | Zoom |
+| Section |      Position      |         Name        |             Email            |       Office Hours      | Office Hour Location |
+| ------- | :----------------: | :-----------------: | :--------------------------: | :---------------------: | :------------------: |
+| All     | Course coordinator |      Julia Peng     |   courses[-at-]stat.ubc.ca   |           n/a           |          n/a         |
+| 002     |     Instructor     |     Daniel Chen     | Daniel Chen[-at-]stat.ubc.ca |  Tuesdays 13:30 - 14:30 |       HENN 200       |
```
Comment (mgyliu) [inline README.md:71]: Is there a typo in your email?

**Review by mgyliu — APPROVED** [review]
lgtm

**Pull request merged**